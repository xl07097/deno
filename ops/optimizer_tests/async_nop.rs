async fn op_void_async() {
  // @test-attr:fast
}
