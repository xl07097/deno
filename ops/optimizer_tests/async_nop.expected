=== Optimizer Dump ===
returns_result: false
has_ref_opstate: false
has_rc_opstate: false
has_fast_callback_option: false
needs_fast_callback_option: false
fast_result: Some(Void)
fast_parameters: [V8Value, I32]
transforms: {}
is_async: true
fast_compatible: true
